### PR TITLE
Dev Plan Changes

### PR DESCRIPTION
**Pull request by TAsif**
First commit - Workflow and generic LaTeX updates

### CODE REVIEW

File: docs/DevelopmentPlan/DevelopmentPlan.tex [hunk 1]
```diff
@@ -14,39 +15,104 @@
...
+\caption{Team Developer Roles} \label{TblDevRoles}
+\begin{tabularx}{\textwidth}{llX}
+\toprule
+\textbf{Team Member} & \textbf{Role}\\
+\midrule
+Talha & DevOps, Full-Stack Developer\\
+Jarrod & Back-End Developer\\
+Madhi & Back-End Developer\\
+Andrew & Front-End Developer\\
+Ali & Front-End Developer\\
```
Comment (TAsif) [inline docs/DevelopmentPlan/DevelopmentPlan.tex]: Give ideas for other roles, if y'all think I need to be more descriptive

File: docs/DevelopmentPlan/DevelopmentPlan.tex [hunk 2]
```diff
@@ -14,39 +15,104 @@
...
 \caption{Revision History} \label{TblRevisionHistory}
 \begin{tabularx}{\textwidth}{llX}
 \toprule
 \textbf{Date} & \textbf{Developer(s)} & \textbf{Change}\\
 \midrule
-Date1 & Name(s) & Description of changes\\
-Date2 & Name(s) & Description of changes\\
-... & ... & ...\\
+Sept 21/22 & Talha & Updating Workflow Plan\\
+Sept 24/22 & Talha & Updating Sections 1-3\\
```
Comment (TAsif) [inline docs/DevelopmentPlan/DevelopmentPlan.tex:29]: I changed the formatting around but if its gonna cost marks cause its different then I'll change back 👀

File: docs/DevelopmentPlan/DevelopmentPlan.tex (hunk 1 above)
Comment (madhi-naga) [reply docs/DevelopmentPlan/DevelopmentPlan.tex]: Should be good enough

File: docs/DevelopmentPlan/DevelopmentPlan.tex [hunk 3]
```diff
@@ -14,39 +15,104 @@
...
+\end{enumerate}
+
+\subsection{Example Documentation Workflow}
+An example documentation workflow will be as follows:
+\begin{enumerate}
+	\item Asynchronous discussion on Discord is done to choose which sections of a documentation to update
+	\item Branch is made for updates and then marked ``Ready for Review" when the owner thinks the documentation has been sufficiently written
+	\item PR is merged
+\end{enumerate}
+
```
Comment (madhi-naga) [inline docs/DevelopmentPlan/DevelopmentPlan.tex:115]: LGTM, there's a couple spots that may need rewording, but I'll add those in myself as I gotta work on the other sections anyway

**Review by madhi-naga — DISMISSED** [review]
LGTM, couple spots that may need rewording, but I'll add those in for my dev plan PR

File: docs/DevelopmentPlan/DevelopmentPlan.tex [hunk 4]
```diff
@@ -14,39 +15,104 @@
...
-\maketitle
+\tableofcontents
+
+\newpage
 
-\wss{Put your introductory blurb here.}
+This documentation details the entire development plan, from team meetings, to the workflow, all the way to the technology Flick Picker will use. 7eam fully understand their responsibilities and knows the flow our application will follow. 
 
 \section{Team Meeting Plan}
+There will be an ad hoc meeting nearly weekly to ensure the team is on the same page, where a time is picked that works for all five team members. The reason for the meeting will also be clearly stated before meeting, and we will start as soon as everyone is in the call.
```
Comment (JarrodColwell) [inline docs/DevelopmentPlan/DevelopmentPlan.tex]: Just say weekly instead of 'nearly weekly'

File: docs/DevelopmentPlan/DevelopmentPlan.tex [hunk 5]
```diff
@@ -14,39 +15,104 @@
...
 \section{Workflow Plan}
+Git Workflow: 
+\begin{itemize}
+	\item \emph{develop} branch will be the single source of truth, where the team reviews changes before they get merged. Thus the \emph{develop} branch will have restricted permissions on it, preventing direct merges without admin overwrite, and only one developer will be the admin, Talha
+	\item Any changes have to be on their branch, and a PR has to be cut with a full green checklist to get it merged into \emph{develop}
+	\item The checklist will grow as development on the application continues, currently, the checklist is a single item, where the PR must have two approvals from the team. The future checklist items are as planned:
+	\begin{itemize}
+		\item Entire test run has to be successful to ensure \emph{develop} is in a healthy state
+		\item Test coverage delta must not be reduced unless redundant tests are taken out
+		\item Spotbugs must pass, enforcing healthy code practices
```
Comment (JarrodColwell) [inline docs/DevelopmentPlan/DevelopmentPlan.tex]: remove spotbugs stuff

File: docs/DevelopmentPlan/DevelopmentPlan.tex [hunk 6]
```diff
@@ -14,39 +15,104 @@
...
+Issue Tracking - \href{https://flickpicker.atlassian.net/jira/software/projects/CAP/boards/1}{Jira}: 
 \begin{itemize}
-	\item How will you be using git, including branches, pull request, etc.?
-	\item How will you be managing issues, including template issues, issue
-	classificaiton, etc.?
+	\item All development changes need a descriptive ticket cut before it is ready for code review. Automation will link the PR to the ticket and vice versa as well
+	\begin{itemize}
+		\item Descriptive means the ticket must have a title and acceptance criteria (AC) to complete the ticket
+	\end{itemize}
+	\item The status of the ticket must be updated on the board. Most importantly, if it is in ``To Do" so multiple developers do not start working on the same feature
```
Comment (JarrodColwell) [inline docs/DevelopmentPlan/DevelopmentPlan.tex]: Last sentence needs some English help

**Review by madhi-naga — APPROVED** [review]

**Review by ali-tab — APPROVED** [review]

**Pull request merged**